doc/rst/usingchapel/chplenv.rst

```diff
@@ -195,8 +195,51 @@ CHPL_*_COMPILER
         pgi                 The PGI compiler suite -- pgcc and pgc++
         =================== ===================================================
 
-   The default for ``CHPL_*_COMPILER`` is llvm, except on linux32 where
-   it is gnu.
+   The default for ``CHPL_HOST_COMPILER`` depends on the value of the
+   corresponding ``CHPL_HOST_PLATFORM`` environment variable:
+
+        +-------------+------------------------------------------------------+
+        | Platform    | Compiler                                             |
+        +=============+======================================================+
+        | cray-xc     | gnu                                                  |
+        |             |                                                      |
```

Comment:
The blank lines in this table seem odd. Are they required by the RST syntax? If not, I'd prefer to delete them. If yes, can we move the compiler name down to the middle line?

Reply:
I addded them in order to get the RST rendering better - it is a way to ask for the words in  one cell to be in different lines

Reply:
Otherwise it looks like
```
cray-xc hpe-
cray-ex
```

Reply:
I moved the Compiler to the middle line and used a longer title + nonbreaking space to avoid the line split on the dashes.